modules/import/csv_file.py

```diff
@@ -71,10 +71,10 @@ def module_run(self):
         for row in self.__values[(1 if has_header else 0):]:
             # creates a dictionary where the keys are the column names and the values are the column values from row
             data = dict(
-                zip(
+                list(zip(
```

Comment:
I found that quite often this casting was unnecessary in the core and removed it. It depends on what's being used with the returned object, but in most cases you can iterate through it like a normal list. This was true in every case for `dict.keys()`.